truss/templates/server.Dockerfile.jinja

```diff
@@ -89,7 +91,18 @@ COPY ./{{ config.model_module_dir }} /app/model
 
 
 {% block run %}
-    {%- if config.live_reload %}
+    {%- if config.docker_server %}
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        curl nginx python3-pip && \
+        rm -rf /var/lib/apt/lists/*
+RUN pip3 install supervisor==4.2.5
```

Comment:
Can we move this to another requirements file or to a constant? I worry that we'll never update the version if it's in the dockerfile here